shared/createAnApp/_default.md

```diff
@@ -1,5 +1,7 @@
-To create an application simply type in a name, select the {{ $device.name }} type from the drop down list and click the create button. You should now be taken to the dashboard of your newly created application:
+An application is a group of devices of the same type that all run the same code. When you provision a device, it is added to a specific application, but can be migrated to another application at any time.
 
-<img src="/img/common/main_dashboard/select_fleet_type.png" width="80%">
+To create an application, simply select the {{ $device.name }} type from the drop-down list, type in a name, and click *CREATE NEW APPLICATION*:
 
-This dashboard may not look like much right now, but this is where you will command and manage a whole fleet of {{ $device.name }}s.
+<img src="/img/common/main_dashboard/select_fleet_type.png" width="40%">
```

Comment:
It would probably be a fair chunk of work creating all the screenshots, but it would be awesome if the image used here could also be paramaterised by `$device` so that e.g. the image at https://docs.resin.io/beaglebone/nodejs/getting-started/#creating-an-application could show the Beaglebone selected in the device-type dropdown :-)

Reply:
It shouldn't be too difficult. I'll see how easy it is to implement with the current set up, and it will definitely be this way once we have our docs generated by contracts.